kie-server-parent/kie-server-controller/kie-server-controller-websocket-client/src/main/java/org/kie/server/controller/websocket/client/WebSocketKieServerControllerImpl.java

```diff
@@ -61,12 +61,16 @@ public WebSocketKieServerControllerImpl() {
         
         this.client = new KieServerMessageHandlerWebSocketClient((WebSocketClient client) -> {
             try {
+                KieServerSetup kieServerSetup = new KieServerSetup();
+                CountDownLatch waitLatch = new CountDownLatch(1);
                 ((KieServerMessageHandlerWebSocketClient) client).sendTextWithInternalHandler(serialize(serverInfo),
-                                                                                              message -> {
-                                                                                                  logger.info("Successfully reconnected");
-                                                                                                  return null;
```

Comment:
@elguardian I think you can probably skip the setup handler and in here just return `return new CommandScriptMessageHandler(context)` that way there is a handler in place capable of handling future requests.